src/SprykerEco/Yves/Unzer/UnzerFactory.php

```diff
@@ -30,6 +36,16 @@ public function createMarketplaceBankTransferFormDataProvider(): StepEngineFormD
         );
     }
 
+    /**
+     * @return \Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface
+     */
+    public function createMarketplaceSfortFormDataProvider(): StepEngineFormDataProviderInterface
```

Comment:
Please, fix the method's name `createMarketplaceSofortFormDataProvider()`

Reply:
Fixed.